arbitrator/include/capabilities_interface.tpp

```diff
@@ -23,38 +23,91 @@
+                            "Failed to retrieve available strategic plugin service topics! Returning empty maneuver responses");
+            return responses;
+        }
+
+        retry_attempt = 0;
+        while (!topics_to_retry.empty() && retry_attempt < MAX_RETRY_ATTEMPTS)
+        {
+            current_topics_to_check = topics_to_retry;
+            topics_to_retry = {};
+            for (const auto & topic : current_topics_to_check)
+            {
+                try {
+                    using std::literals::chrono_literals::operator""ms;
 
-                const auto client = nh_->create_client<carma_planning_msgs::srv::PlanManeuvers>(topic);
-                RCLCPP_DEBUG_STREAM(rclcpp::get_logger("arbitrator"), "found client: " << topic);
+                    const auto client = nh_->create_client<carma_planning_msgs::srv::PlanManeuvers>(topic);
+                    RCLCPP_DEBUG_STREAM(rclcpp::get_logger("arbitrator"), "found client: " << topic);
 
-                const auto response = client->async_send_request(msg);
+                    const auto response = client->async_send_request(msg);
 
-                switch (const auto status{response.wait_for(500ms)}; status) {
-                    case std::future_status::ready:
-                        responses.emplace(topic, response.get());
-                        break;
-                    case std::future_status::deferred:
-                    case std::future_status::timeout:
-                        RCLCPP_WARN_STREAM(rclcpp::get_logger("arbitrator"), "failed...: " << topic);
+                    switch (const auto status{response.wait_for(500ms)}; status) {
+                        case std::future_status::ready:
+                            responses.emplace(topic, response.get());
+                            break;
+                        case std::future_status::deferred:
+                        case std::future_status::timeout:
+                            RCLCPP_WARN_STREAM(rclcpp::get_logger("arbitrator"), "failed...: " << topic);
+                    }
+                    if (retry_attempt > 0)
+                    {
+                        RCLCPP_INFO_STREAM(rclcpp::get_logger("arbitrator"),"Service call to: " << topic << ", successfully finished after retrying: " << retry_attempt);
+                    }
+
+                } catch(const rclcpp::exceptions::RCLError& error) {
+                    RCLCPP_WARN_STREAM(rclcpp::get_logger("arbitrator"),
+                        "Cannot make service request for service '" << topic << "': " << error.what() <<". So retrying, attempt no: " << retry_attempt);
+                    // Fast repeated calls may fail and on average in one of our environment was failing at 70ms repeated calls
+                    // https://github.com/ros2/rmw_fastrtps/issues/46
+                    std::chrono::milliseconds duration(100);
+                    std::this_thread::sleep_for(duration);
```

Comment:
These two lines can be combined:
```cpp
std::this_thread::sleep_for(std::chrono::milliseconds{100});
```
or 
```cpp
using std::chrono::literals::operator""ms;  // May not be needed if declared elsewhere
std::this_thread::sleep_for(100ms);
```

Reply:
Removed this section because it did not fix or improve anything